docs/Running-Flink-on-Tachyon.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: global
+title: Running Apache Flink on Tachyon
+---
+
+This guide describes how to get Tachyon running with [Apache Flink](http://flink.apache.org/), so that you can easily work with files stored in Tachyon.
+
+# Prerequisites
+
+The prerequisite for this part is that you have
+[Java](https://github.com/amplab/tachyon/wiki/Java-setup/). We also assume that you have set up
+Tachyon and Flink in accordance to these guides [Local Mode](Running-Tachyon-Locally.html) or
+[Cluster Mode](Running-Tachyon-on-a-Cluster.html). 
+
+Plese find the guides for setting up Flink on the [Apache Flink website](http://flink.apache.org/).
+
+
+# Configuration
+
+Apache Flink allows to use Tachyon through a generic file system wrapper for Hadoop file systems. Therefore, the configuration of Tachyon is done mostly in Hadoop configuration files.
+
+#### Set property in `core-site.xml`
+
+If you have a Hadoop setup next to the Flink installation, add the following property to the `core-site.xml` configuration file:
```

Comment:
Sadly Flink can not re-use the Hadoop code for loading the classes because the methods are different between the Hadoop1 and the hadoop2 release.
To avoid complicated maven module constructions, I just did the classloading myself.